dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -10,11 +10,41 @@
 
 %p
   We’re excited to continue to work together to bring computer science to your classroom.
+  - if @workshop.local_summer? && @is_reminder
+    Please take a moment to complete the pre-course survey and review the
+    following logistical details about your workshop.
   If you have any questions about your workshop, reach out to your workshop organizer directly:
   = @workshop.organizer.name
   at
   = mail_to(@workshop.organizer.email) + '.'
 
+- if @workshop.local_summer? && @is_reminder
+  %h3
+    Pre-Course Survey
+  %p
+    Before starting your Summer Workshop,
+    = link_to "please take this survey", "http://code.org/csp-pre-survey"
```

Comment:
Do we want to hard-code the url on `code.org` or use something like `CDO.code_org_url('/csp-pre-survey', CDO.default_scheme)`